webpack.config.js

```diff
@@ -184,6 +185,7 @@ const config = (module.exports = {
         process.env.MB_EDITION === "ee"
           ? ENTERPRISE_SRC_PATH + "/overrides"
           : SRC_PATH + "/lib/noop",
+      "embedding-sdk": SDK_SRC_PATH,
```

Comment:
added this so we can use `embedding-sdk/...` for the paths. just a nice little QoL thing for myself

Reply:
yeah makes sense. I only put it there because I was worried that eslint wouldn't pick it up

Reply:
Ok I tried moving the alias to the `embedding-sdk.config.js`. It looks like eslint is only picking up the aliases within the main webpack config file, and from what I see on `eslint-import-resolver-webpack`'s documentation, there's not much we can do besides put the `embedding-sdk` in the main webpack config. You ok with keeping it there? I don't really see a downside to keeping that alias in the main config

Reply:
ok cool then. I didn't know that has something to do with our ESLint setup.